web/src/components/Section.js

```diff
@@ -53,7 +53,7 @@ const Subsection = ({ children, id, nested, resource }) => {
         </h3>
       )}
       {nested && (
-        <h4 id={id} className="ds-h3">
+        <h4 id={id} className="ds-h4">
```

Comment:
Note to @hbillings, this changes the styling for all of the sections.  Is that the intended result?  If not I'll need to go back in and tweak it.